### ISSUE TITLE
Rule suggestion: Enforce padding between class functions

### ISSUE THREAD

**Issue by LinusU** [standard/standard#438]
I would like to propose a rule that enforces consistent spacing between class functions.

The following patterns would be considered problems:

``` javascript
class Test1 {
  constructor () {
    // ...
  }
  otherFunc () {
    // ...
  }
}

class Test2 {
  constructor () {
    // ...
  }


  otherFunc () {
    // ...
  }
}
```

The following pattern would be considered valid:

``` javascript
class Test3 {
  constructor () {
    // ...
  }

  otherFunc () {
    // ...
  }
}
```

**Comment by yoshuawuyts** [comment]
sure, seems reasonable

**Comment by feross** [comment]
There's no eslint core rule for this? @LinusU can you propose it on the eslint repo?

**Comment by LinusU** [comment]
eslint/eslint##5949

**Comment by LinusU** [comment]
I also created a working rule 🙌 

eslint/eslint#5950

**Comment by feross** [comment]
@LinusU Nice work! Please post a comment here / send a PR when that's merged and released in ESLint so we can add this rule to `standard`.

**Comment by LinusU** [comment]
Will do :ok_hand:

**Comment by feross** [comment]
@LinusU Looks like progress is going well on this rule -- it's almost there! I'm going to close this issue since it's already being tracked in the ESLint repo and there's nothing for us to do here.

Please open a new issue or PR when this rule is actually added to ESLint and I'll enable it. If you forget, no worries either. Every time I update ESLint, I evaluate every new rule for inclusion in `standard` so I will see it.

**Comment by LinusU** [comment]
https://github.com/eslint/eslint/commit/ee99876ccc76585b0af2c8673d7eb25bb0b972fa

It has finally landed 🚀 🎉

**Comment by LinusU** [comment]
Landed in eslint 4.9.0 🎉

**Comment by feross** [comment]
Let's include this in `standard` 13.

Ecosystem impact: only one failing package (`webtorrent`) and was automatically fixable.
